### DISCUSSION TITLE
How we can access github profile link

### DISCUSSION THREAD

**Discussion by RAJAT123455** [github/india#189] [GitHub Platform]
how we can submit a Github profile link

**Comment by TheSuranaverse** [comment]
@RAJAT123455 is your doubt solved?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by FRTMentor** [comment]
Hi @RAJAT123455,

You can submit in the orientation section of the below link.
And profile link should be unique and shouldn't be redundant over the other submissions.
https://futurereadytalent.in/learning
